gcp_deepvariant_runner.py

```diff
@@ -277,6 +335,22 @@ def _gcs_object_exist(gcs_obj_path):
     return False
 
 
+def _get_gcs_object_size(gcs_obj_path):
+  """Returns the size of the given GCS object.
+
+  Args:
+    gcs_obj_path: (str) a path to an obj on GCS.
+  """
+  storage_client = storage.Client()
+  bucket_name = _get_gcs_bucket(gcs_obj_path)
+  obj_name = _get_gcs_relative_path(gcs_obj_path)
+  bucket = storage_client.get_bucket(bucket_name)
```

Comment:
Does this and the below throw exceptions? If yes either catch or declare in the comment.